analysis/statistical_analyzer.py

```
@@ -0,0 +1,572 @@
+            'std': float(series.std()),
+            'min': float(series.min()),
+            'max': float(series.max()),
+            'q25': float(series.quantile(0.25)),
+            'q75': float(series.quantile(0.75)),
+        }
+        
+        # Mode (can be multiple values)
+        try:
+            mode_result = stats.mode(series, keepdims=True)
+            stats_dict['mode'] = float(mode_result.mode[0])
+        except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)